### DISCUSSION TITLE
Discussion Thread about what project idea we should implement in this class

### DISCUSSION THREAD

**Discussion by saipavan10-git** [saipavan10-git/COVID-19-Vaccination-Portal#2] [Ideas]
This discussion tracks everything in our first meeting

**Comment by mayichong** [comment]
I have two projects ideas. 
The first one is Real estate transaction (rental, purchase and sell) web application, such as Zillow, Trulia, etc. Users can register and log into the system to check real estate properties. There can be user interactions, such as chat, email and comment system.

The second one is Blog interaction. Again, users can register and log into the system. Users can write their own personal stories or ideas on their webpage. Other users can take a look at their stories/ideas and comment down below. The format can be words, pictures, videos, etc.

**Comment by saipavan10-git** [comment]
My Project Ideas are :
🥇 COVID 19 Vaccination Portal Similar to COWIN in India. A website where we can book our appointments, see our vaccination status, number of days from next dose and a verifiable certificate for vaccination
🔢 Note Pad app
🔢 Pre Ordering Food in Events like concerts, games, movies, etc.
🔢 Pill Remainder app for elderly
🔢 Password manager 
🔢 Tickemaster Lite

**Comment by Vtanneru** [comment]
Ticketmaster Lite

1.Single App for all ticketing Purposes of various Transport,Entertainment and Events
2.Ticket Selling with also a facility to reselling the ticket
3.Location and real time navigator to the location
4.Real time purchases of various food at the entertainement and event spots
5.Customer care service helping with various issues and on call bookings.
